src/views/partials/presentation-table.hbs

```diff
@@ -1,13 +1,26 @@
-<tbody id="search-results">
+<link rel="stylesheet" type="text/css" href="/css/presentation-style.css" />
+{{#each presentations}}
+<div data-testId="presentation-row">
+    <div class="container">
+        <div class="homepage">
+            <ul>
+                <li>
+                    <a href="/presentations/{{id}}">
```

Comment:
Moved link to wrap just the topic name. Otherwise favoriting didn't work without additional effort.